statics/js/components/graph-layout.js

```diff
@@ -402,6 +409,7 @@ TopologyGraphLayout.prototype = {
   },
 
   onNodeAdded: function(node) {
+    if (this.hidden(node)) return;
```

Comment:
It is used by onNodeUpdated. But the real solution is probably to add if(this.hidden(node)) return to both onNodeUpdated and onNodeDeleted. We want to have a tunable set of nodes completely ignored by a renderer. To make it complete I have also added it to onEdgeDeleted (remove if either end is hidden).